rook/deploy-rook.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+helm repo add rook-release https://charts.rook.io/release
+
+namespace="rook-ceph"
+release_name="rook-ceph"
+chart="rook-release/rook-ceph"
+chart_version="v1.5.3"
+
+# Install rook operator
+kubectl create namespace $namespace
+helm install --namespace $namespace $release_name $chart \
```

Comment:
In the same spirit we should change this to
```suggestion
helm update --install --namespace $namespace $release_name $chart \
```

Reply:
:+1: